src/routes/_surfaces/workspace/explore/MetricsTimeSeriesCharts.svelte

```diff
@@ -1,14 +1,28 @@
-<script>
+<script lang="ts">
   import MetricsExploreTimeChart from "$lib/components/leaderboard/MetricsExploreTimeChart.svelte";
-  export let metricsDefId;
+  import type { Readable } from "svelte/store";
+  import type { MeasureDefinitionEntity } from "$common/data-modeler-state-service/entity-state-service/MeasureDefinitionStateService";
+  import { getMeasureById } from "$lib/redux-store/measure-definition/measure-definition-readables";
+
+  export let metricsDefId: string;
+  export let measureId: string;
+  export let index: number;
+
+  let measure: Readable<MeasureDefinitionEntity>;
+  $: measure = getMeasureById(measureId);
 </script>
 
-<div>
-  <div class="grid grid grid-flow-col">
-    <div class="big-number">
-      <h2>Place for name</h2>
-      <div />
+{#if $measure}
+  <div>
+    <div class="grid grid grid-flow-col">
+      <div class="big-number">
+        <h2>{$measure.label ?? $measure.sqlName ?? $measure.expression}</h2>
+        <div />
+      </div>
+      <MetricsExploreTimeChart
+        {metricsDefId}
+        yAccessor={$measure.sqlName ?? `measure_${index}`}
```

Comment:
```suggestion
        yAccessor={$measure.sqlName && $measure.sqlName?.length !== 0
          ? $measure.sqlName
          : `measure_${index}`}
```

Reply:
What you have her doesn't work if `$measure.sqlName` is an empty string, which turns out to be true when `sqlName` is not defined in the metrics config.